charts/am-org-role-mapping-service/values.preview.template.yaml

```diff
@@ -30,7 +30,9 @@ java:
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CASE_WORKER_REF_APP_URL: http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal
     JUDICIAL_REF_APP_URL: http://rd-judicial-api-aat.service.core-compute-aat.internal
-    ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
```

Comment:
Note: revert back to AAT RAS instance prior to master merge

Reply:
Resolved in 929cbbdbda46f5be83a9db9d329ba1faa43eedbe.